SPECS/fluent-bit/fluent-bit.spec

```diff
@@ -28,34 +39,50 @@ Development files for %{name}
 %setup -q
 
 %build
-cd build
-cmake -DCMAKE_INSTALL_PREFIX=%{_prefix} ..
-make %{?_smp_mflags}
+	
+%cmake\
+    -DCMAKE_BUILD_TYPE=RelWithDebInfo\
+    -DFLB_EXAMPLES=Off\
+    -DFLB_OUT_SLACK=Off\
+    -DFLB_IN_SYSTEMD=On\
+    -DFLB_OUT_TD=Off\
+    -DFLB_OUT_ES=Off\
+    -DFLB_SHARED_LIB=On\
+    -DFLB_TESTS_RUNTIME=On\
+    -DFLB_TESTS_INTERNAL=Off\
+    -DFLB_RELEASE=On\
+    -DFLB_DEBUG=Off\
+    -DFLB_TLS=On
+ 
+%cmake_build
 
 %install
-cd build
-make install DESTDIR=%{buildroot}
+%cmake_install
 
 %files
 %license LICENSE
 %doc README.md
-%exclude /usr/src/debug
+%exclude %{_prefix}/src/debug
 /lib/systemd/system/fluent-bit.service
 %{_bindir}/*
-/usr/etc/fluent-bit/*
+%{_prefix}%{_sysconfdir}/fluent-bit/*
 
 %files devel
 %{_includedir}/*
-/usr/lib64/fluent-bit/*.so
+%{_lib}/fluent-bit/*.so
```

Comment:
I'd double-check that this works. If I remember correctly, `%{_lib}` translates to `lib` and we should probably use `%{_libdir}` instead. I **think** this has been addressed by Thomas a while ago, so may try refreshing your local linter configuration using his fork.

Reply:
Yea, just by expanding the macro it is clear this is wrong, thanks for catching this! I'll have to be more skeptical of the linter suggestions in the future.